aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -119,6 +199,46 @@ module aptos_framework::staking_config {
         config.voting_power_increase_limit
     }
 
+    /// Return the rewards rate of a epoch in the format of (nominator, denominator).
+    public fun calculate_and_save_latest_epoch_rewards_rate(): FixedPoint64 acquires StakingRewardsConfig {
```

Comment:
Can this be private?

Reply:
This one cannot be private. `stake.move` needs to call it and `staking_config.move` is not a friend of `stake.move`.